scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
Does that support both x64 and x86? (and ARMv7 support too would be fantastic :smile: )